tests/setup/saveStorageState.js

```diff
@@ -6,16 +6,22 @@ import { USER1_STORAGE_STATE_PATH } from '../../data/constants';
 setup('Login and save storage state', async ({ page }) => {
     const welcomePage = new WelcomePage(page);
     await welcomePage.navigate();
-    await welcomePage.clickSignInButton();
-    
-    // Заполнение формы логина
-    await page.fill('input[name="email"]', USERS.USER1.email);
-    await page.fill('input[name="password"]', USERS.USER1.password);
-    await page.click('button:has-text("Login")');
-    
-    // Ожидание перехода на страницу гаража
-    await page.waitForURL('/panel/garage');
-    
+
+    // Проверяем, находится ли пользователь на странице /panel/garage
+    const currentUrl = await page.url();
+    if (!currentUrl.includes('/panel/garage')) {
+        // Если пользователь не на странице гаража, предполагаем, что он не залогинен.
+        await welcomePage.clickSignInButton();
+        
+        // Заполнение формы логина
+        await page.fill('input[name="email"]', USERS.USER1.email);
+        await page.fill('input[name="password"]', USERS.USER1.password);
+        await page.click('button:has-text("Login")');
+        
+        // Ожидание перехода на страницу гаража
+        await page.waitForURL('/panel/garage');
+    }
```

Comment:
user should not be on garage page

you have to login each time and save new state